src/ansys/aedt/core/maxwell.py

```diff
@@ -976,21 +1041,21 @@ def assign_voltage(self, assignment, amplitude=1, name=None):
 
         Examples
         --------
-
         Create a region in Maxwell 2D and assign voltage to its edges.
+
         >>> from ansys.aedt.core import Maxwell2d
         >>> m2d = Maxwell2d(version="2025.1", solution_type="ElectrostaticZ")
         >>> region_id = m2d.modeler.create_region(pad_value=[500, 50, 50])
         >>> voltage = m2d.assign_voltage(assignment=region_id.edges, amplitude=0, name="GRD")
```

Comment:
@nunesanderson @anur7 stupid question but what's the point of assigning a voltage excitation to region's edges?